api/app/src/Entity/CourtOrder.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace App\Entity;
+
+use App\Entity\Traits\CreateUpdateTimestamps;
+use Doctrine\ORM\Mapping as ORM;
+use JMS\Serializer\Annotation as JMS;
+
+/**
+ * Court Orders for clients.
+ *
+ * @ORM\Table(name="court_order")
+ *
+ * @ORM\Entity()
+ *
+ * @ORM\HasLifecycleCallbacks()
+ */
+class CourtOrder
+{
+    use CreateUpdateTimestamps;
+
+    /**
+     * @var int
+     *
+     * @JMS\Type("integer")
+     *
+     * @ORM\Column(name="id", type="integer", nullable=false)
+     *
+     * @ORM\Id
+     *
+     * @ORM\GeneratedValue(strategy="IDENTITY")
+     *
+     * @ORM\SequenceGenerator(sequenceName="court_order_id_seq", allocationSize=1, initialValue=1)
+     */
+    private $id;
+
+    /**
+     * @var int
+     *
+     * @JMS\Type("integer")
+     *
+     * @ORM\Column(name="court_order_uid", type="integer", nullable=false, unique=true)
+     */
+    private $courtOrderUid;
+
+    /**
+     * @var string
+     *
+     * @JMS\Type("string")
+     *
+     * @ORM\Column(name="type", type="string", length=10, nullable=false)
+     */
+    private $type;
+
+    /**
+     * @var bool
+     *
+     * @JMS\Type("boolean")
+     *
+     * @ORM\Column(name="active", type="boolean", options = { "default": true })
+     */
+    private $active;
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @return $this
+     */
+    public function setId(int $id)
+    {
+        $this->id = $id;
+
+        return $this;
+    }
+
+    public function getCourtOrderUid(): int
+    {
+        return $this->courtOrderUid;
+    }
+
+    /**
+     * @return $this
+     */
+    public function setCourtOrderUid(int $courtOrderUid)
```

Comment:
I think we can remove the annotation from the set methods and use the return types syntax like the get methods.

`public function setCourtOrderUid(int $courtOrderUid): CourtOrder`